ast/parser.go

```diff
@@ -2569,3 +2558,33 @@ func (p *Parser) futureImport(imp *Import, allowedFutureKeywords map[string]toke
 		p.s.s.AddKeyword(kw, allowedFutureKeywords[kw])
 	}
 }
+
+func (p *Parser) regoV1Import(imp *Import, allowedFutureKeywords map[string]tokens.Token) {
+	path := imp.Path.Value.(Ref)
+
+	if len(path) == 1 || !path[1].Equal(regoV1CompatibleRef[1]) || len(path) > 2 {
+		p.errorf(imp.Path.Location, "invalid import, must be `%s`", regoV1CompatibleRef)
+		return
+	}
+
+	if imp.Alias != "" {
+		p.errorf(imp.Path.Location, "`rego` imports cannot be aliased")
+		return
+	}
+
+	kwds := make([]string, 0, len(allowedFutureKeywords))
```

Comment:
We could simply use `futureKeywords` here, and drop the `allowedFutureKeywords` arg.

Reply:
Updated.